src/components/executive-education-2u/UserEnrollmentForm.jsx

```diff
@@ -198,18 +199,19 @@ const UserEnrollmentForm = ({
             <Card.Body>
               <Card.Section>
                 <h3 className="mb-2">Course enrollment information</h3>
-                <Alert
-                  variant="danger"
-                  className="mb-4.5"
-                  show={!!formSubmissionError}
-                  onClose={() => setFormSubmissionError(undefined)}
-                  dismissible
-                >
-                  <p>
-                    An error occurred while sharing your course enrollment information. Please try again.
-                  </p>
-                </Alert>
-
+                {formSubmissionError && !isDuplicateOrder(formSubmissionError) && (
+                  <Alert
+                    variant="danger"
+                    className="mb-4.5"
+                    show={!!formSubmissionError}
```

Comment:
I believe we should always render the `Alert` component, but ensure the `show` prop includes the conditional logic for `formSubmissionError && !isDuplicateOrder(formSubmissionError)` instead.